devices_test.go

```diff
@@ -39,7 +39,7 @@ var (
 func waitDeviceActive(t *testing.T, c *Client, id string) *Device {
 	// 15 minutes = 180 * 5sec-retry
 	for i := 0; i < 180; i++ {
-		<-time.After(5 * time.Second)
+		<-time.After(15 * time.Second)
```

Comment:
Do you intend to increase the timeout? 
https://github.com/packethost/packngo/pull/127/files#diff-1bce0d89d0df728a0b209da06d68a7d30cb782b0aa4fe7e7307038adc9e51debR40

Reply:
no, I'll remove it

Reply:
I think that (2 years ago) I had to increase the timeout because the reinstall takes really long time. It takes longer than the 5 minutes with `deprovision_fast: true`, I didn't try without. But maybe it's up to the linking library to handle the timeout..